js/jsmodules.go

```diff
@@ -15,28 +15,30 @@ import (
 	"go.k6.io/k6/js/modules/k6/metrics"
 	"go.k6.io/k6/js/modules/k6/ws"
 
+	"github.com/grafana/xk6-browser/browser"
 	"github.com/grafana/xk6-redis/redis"
 	"github.com/grafana/xk6-timers/timers"
 	expws "github.com/grafana/xk6-websockets/websockets"
 )
 
 func getInternalJSModules() map[string]interface{} {
 	return map[string]interface{}{
-		"k6":                         k6.New(),
-		"k6/crypto":                  crypto.New(),
-		"k6/crypto/x509":             x509.New(),
-		"k6/data":                    data.New(),
-		"k6/encoding":                encoding.New(),
-		"k6/execution":               execution.New(),
-		"k6/experimental/redis":      redis.New(),
-		"k6/experimental/websockets": &expws.RootModule{},
-		"k6/experimental/timers":     timers.New(),
-		"k6/experimental/tracing":    tracing.New(),
-		"k6/net/grpc":                grpc.New(),
-		"k6/html":                    html.New(),
-		"k6/http":                    http.New(),
-		"k6/metrics":                 metrics.New(),
-		"k6/ws":                      ws.New(),
+		"k6":                          k6.New(),
+		"k6/crypto":                   crypto.New(),
+		"k6/crypto/x509":              x509.New(),
+		"k6/data":                     data.New(),
+		"k6/encoding":                 encoding.New(),
+		"k6/execution":                execution.New(),
+		"k6/experimental/redis":       redis.New(),
+		"k6/experimental/websockets":  &expws.RootModule{},
+		"k6/experimental/timers":      timers.New(),
+		"k6/experimental/tracing":     tracing.New(),
+		"k6/experimental/xk6-browser": browser.New(),
```

Comment:
Should it be just a `k6/experimental/browser`? :thinking:

Reply:
Thanks, resolved in c0a5be61f10deb54c756e638907b353cf0cd492e